hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/DefaultRecordStore.java

```diff
@@ -837,14 +837,26 @@ public Object replace(Data key, Object update) {
         long now = getNow();
```

Comment:
i see a TODO [here](https://github.com/hazelcast/hazelcast/blob/fda3b03012535205966755059e6f61999d1fa20e/hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/DefaultRecordStore.java#L833)
can you please also remove it?